build/DependencyVersions.props

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <MicrosoftAspNetCoreAllPackageVersion>2.1.0-preview2-30475</MicrosoftAspNetCoreAllPackageVersion>
```

Comment:
Are these two ever going to differ? If not, can we make one of these depend on the other?

Reply:
I don't foresee these being different form now until RTM. There is a chance (though small) that they may different once we start patching 2.1.x.

Reply:
Good suggestion. That's good enough for now IMO.